src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -388,6 +388,10 @@ pub struct Args {
     /// The PostgreSQL URL for the storage stash.
     #[clap(long, env = "STORAGE_STASH_URL", value_name = "POSTGRES_URL")]
     storage_stash_url: String,
+    /// The directory of instance storage. If the process orchestrator
+    /// is being used, this can be a relative path.
+    #[clap(long, env = "INSTANCE_DIRECTORY", value_name = "PATH")]
+    instance_directory: Option<PathBuf>,
```

Comment:
Agree we should try to find a better name than this! It is EC2 specific in its current incarnation, and "instance" is already a massively overloaded term. Perhaps `ephemeral_directory` or `scratch_directory` or somesuch.

Reply:
Actually, I'm pretty sure this needs to be process orchestrator specific. The Kubernetes orchestrator will conjure up a scratch directory upon request, by creating a volume mount of the right type.